### PR TITLE
Small fixed on polling/ added leaderboard

### PR DESCRIPTION
**Pull request by markmiw**

### CODE REVIEW

File: database/schema.sql [hunk 1]
```diff
@@ -32,6 +32,8 @@ CREATE TABLE chat (
   id SERIAL PRIMARY KEY NOT NULL,
   date BIGINT NOT NULL,
   message VARCHAR(500) NOT NULL,
+  country VARCHAR(50) NOT NULL,
+  username VARCHAR(256) NOT NULL,
```
Comment (corneliusrenken) [inline database/schema.sql:36]: why not just use userID to fetch this data?

File: server/routes.js [hunk 2]
```diff
@@ -1,7 +1,13 @@
 const router = require('express').Router();
 
 const { slots } = require('./controllers/slots');
+<<<<<<< HEAD
+const {
+  getUser, getLeaderboard, getCountry, getGlobalChat, postGlobalChat,
+} = require('../database/controllers');
+=======
 const roulette = require('./controllers/roulettecontrollers.js');
+>>>>>>> 1964d9395a0110862d0def478b049cd2432ca9f8
```
Comment (corneliusrenken) [inline server/routes.js]: unresolved merge conflict? Or do we keep everything

File: database/schema.sql [hunk 3]
```diff
@@ -114,5 +116,11 @@ INSERT INTO RouletteNums(id, evenOdd, color, rangeOf12, firstHalf, numRow) VALUE
...
+<<<<<<< HEAD
+-- -- psql postgres postgres < /Users/.... rouletteSchema.sql;
+-- GRANT ALL PRIVILEGES ON ALL TABLES TO postgres;
+-- GRANT ALL PRIVILEGES ON ALL SEQUENCES IN SCHEMA luckylarrys TO postgres;
+=======
 -- psql postgres postgres < /Users/.... rouletteSchema.sql;
--- psql -U postgres \Users\J\Documents\Projects\LUCKYLARRYS\database\schema.sql
\ No newline at end of file
+-- psql -U postgres \Users\J\Documents\Projects\LUCKYLARRYS\database\schema.sql
+>>>>>>> 1964d9395a0110862d0def478b049cd2432ca9f8
```
Comment (corneliusrenken) [inline database/schema.sql:126]: unresolved merge conflict? Or do we keep everything

**Review by corneliusrenken — APPROVED** [review]
Looks good 😎  Left a few minor comments

File: database/schema.sql (hunk 1 above)
Comment (markmiw) [reply database/schema.sql:36]: i figured for username, we can reduce the amount of fetching/requests if we just post and get with the username. If a username changes, i figured we wouldn't need to update all messages to that new username.

File: server/routes.js (hunk 2 above)
Comment (markmiw) [reply server/routes.js]: weird. that unresolved conflict didn't notify me.

File: database/schema.sql (hunk 3 above)
Comment (markmiw) [reply database/schema.sql:126]: just fixed with latest push

**Review by Jesse132 — APPROVED** [review]
Looks good.

File: client/src/components/sidebar/friends/FriendInput.jsx [hunk 4]
```diff
@@ -38,6 +38,8 @@ const FriendInputContainer = styled('form')`
   padding: 0 50px;
   display: flex;
   flex-direction: row;
+  border-radius: 7px;
+  padding-bottom: 5xpx;
```
Comment (xgmak94) [inline client/src/components/sidebar/friends/FriendInput.jsx:42]: 5xpx

**Pull request merged**